tests/setup_test.go

```diff
@@ -3664,3 +3671,168 @@ func TestExportImport(t *testing.T) {
 		t.Fatalf("expected 0 orgs got: %d", len(orgs))
 	}
 }
+
+func TestRunEventsNotification(t *testing.T) {
+	tests := []struct {
+		name                   string
+		config                 string
+		expectedRunResult      rstypes.RunResult
+		expectedRunPhase       rstypes.RunPhase
+		expectedRunPhaseEvents []rstypes.RunPhase
+	}{
+		{
+			name: "test run result success",
+			config: `
+			{
+			  runs: [
+			    {
+			      name: 'run01',
+			      tasks: [
+			        {
+			          name: 'task01',
+			          runtime: {
+			            containers: [
+			              {
+			                image: 'alpine/git',
+			              },
+			            ],
+			          },
+			          steps: [
+			            { type: 'run', command: 'env' },
+			          ],
+			        },
+			      ],
+			    },
+			  ],
+			}
+			`,
+			expectedRunResult:      rstypes.RunResultSuccess,
+			expectedRunPhase:       rstypes.RunPhaseFinished,
+			expectedRunPhaseEvents: []rstypes.RunPhase{rstypes.RunPhaseQueued, rstypes.RunPhaseRunning, rstypes.RunPhaseRunning, rstypes.RunPhaseFinished},
+		},
+		{
+			name: "test run result failed",
+			config: `
+			{
+			  runs: [
+			    {
+			      name: 'run01',
+			      tasks: [
+			        {
+			          name: 'task01',
+			          runtime: {
+			            containers: [
+			              {
+			                image: 'alpine/git',
+			              },
+			            ],
+			          },
+			          steps: [
+			            { type: 'run', command: 'fail test' },
```

Comment:
you should use something clearer like `/bin/false` (command) or just `false` (shell built in)